.github/workflows/ci.yaml

```
@@ -64,10 +64,12 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Install Helm Unittest Plugin
-        run: helm plugin install https://github.com/quintush/helm-unittest
+        run: helm plugin install https://github.com/helm-unittest/helm-unittest
 
-      - name: Run Helm Unittest
-        run: helm unittest charts/cadc-tap
+      - name: Run Helm Unittest for all applications and charts with a `tests` directory
+        run: |
+          dirs=$(find {applications,charts} -maxdepth 2 -mindepth 2 -type d -name tests -exec dirname {} \;)
+          helm unittest $dirs
```

```suggestion
          helm unittest {applications,charts}/*/tests
```
will work now that you're adding tests for mobu so there's at least one application with tests and the first wildcard will be expanded, I think.

---

That doesn't work because the dir with `Chat.yaml` is the dir that needs to passed to `helm unittest`, which is the _parent_ of the tests directory.  There very well may be some `bash` wizardry to get this to work that's nicer than my pages-long find command.

Another option is to pass all of the application and chart dirs with `helm unittest {applications,charts}/*/`, but that has output for all dirs, even if they don't have subdirs, and the results for individual dirs are not moved to the bottom :(

---

...but `helm unittest {applications,charts}/*/tests/..` works!

---

Oh, that's what I missed! How annoying. That's a nice workaround; I probably would have gone for:
```
    helm unittest $(echo {applications,charts}/*/tests} | sed 's%/tests%%g')
 ```
 but that's still pretty ugly and the find is honestly probably better than that.